ccd-definition/AuthorisationCaseField/AuthorisationCaseField-JO-nonprod.json

```diff
@@ -1,8 +1,129 @@
 [
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseFieldID": "joDefendantName1",
+    "UserRoles": [
+      "caseworker-civil-solicitor",
+      "caseworker-civil-admin",
+      "caseworker-civil-staff"
+    ],
+    "CRUD": "CRU"
```

Comment:
if these are going to be fields that are shown only on the judgement tab, can they be R only?

Reply:
yes, think so. Will update them to R.